### PR TITLE
Publish rust-code-analysis automatically on crates.io

### PR DESCRIPTION
**Pull request by Luni-4**
This PR publishes `rust-code-analysis` automatically.

Thanks in advance for your review! :)

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -241,14 +241,14 @@ tasks:
...
-            image: "mozilla/taskboot:0.2.5"
+            image: "mozilla/taskboot:0.2.7"
             env:
               WINDOWS_BIN: rust-code-analysis-cli.exe
             command:
                 - "/bin/sh"
                 - "-cx"
                 - "taskboot retrieve-artifact --output-path=. \
-                            --artifacts=$WINDOWS_BIN:public/$WINDOWS_BIN &&
+                            --artifacts=public/$WINDOWS_BIN &&
```
Comment (marco-c) [inline .taskcluster.yml:251]: I don't remember, did we change the `retrieve-artifact` command recently?

Anyway, this change seems unrelated to this PR, so please open a separate PR with it.

Comment (Luni-4) [reply .taskcluster.yml:251]: Yep, we [did](https://github.com/mozilla/task-boot/commit/34c6274fd1af09b0bce163ec4649ab0deca245e4). 
I thought I could use the same PR to update everything :P
But you're right, I'm going to open a new PR

Comment (marco-c) [reply .taskcluster.yml]: Better to have each PR be a different set of logic changes (or each commit in a PR, but that's harder to handle as sometimes you'd have to rewrite history).

Comment (Luni-4) [reply .taskcluster.yml]: I agree, but since they were both related to `TaskCluster`,I thought I could have used two different commits

**Comment by marco-c** [comment]
It's a pity that now the `deploy_release` task will have to perform a build, but that's OK.

**Comment by Luni-4** [comment]
> It's a pity that now the `deploy_release` task will have to perform a build, but that's OK.

I know, but the entire process happens once in a while
